sources/tools/Stride.SamplesTestServer/SamplesTestServer.cs

```diff
@@ -434,7 +434,7 @@ protected override async void HandleClient(SimpleSocket clientSocket, string url
                 await tester.TesterSocket.Send(new ScreenshotStored());
             });
 
-            Task.Run(async () =>
+            await Task.Run(async () =>
```

Comment:
I don't think we should await it here. So to sum up, in all places where `Task.Run` is used to spawn a thread with a message loop, we should just use `_ = ` to ignore the value and suppress the warning.

Reply:
Ah, that's a good, safe solution I should have thought of.  Will update.

Reply:
Turns out we even made a method to ignore this warning, extension method `Forget()`:

https://github.com/stride3d/stride/blob/6a2e29b6aa4aeaf852631f9466b05b201d182142/sources/core/Stride.Core.Design/Extensions/TaskExtensions.cs#L14

You can see one example usage of it here:
https://github.com/stride3d/stride/blob/7e836297cb5930c01e6dfa0183e7f3cc64748fb6/sources/editor/Stride.Core.Assets.Editor/Components/DebugTools/UndoRedo/DebugUndoRedoViewModel.cs#L49

Please be very careful to not add any await in this PR, it is (usually) skipped in some specific places on purpose and might lead to lot of unexpected block, hang or other undesired side-effect.